### PR TITLE
only push message when error

### PR DESCRIPTION
**Pull request by JingLin0**

### CODE REVIEW

File: tap_tool/__init__.py [hunk 1]
```diff
@@ -122,16 +122,6 @@ def publish_msg(slack, job_failed, company, tap_runtime):
                 color="danger",
             )
         sys.exit(1)
-    else:
-        success_msg = f"All data for {company} were synced successfully!"
-        logging.info(success_msg)
```
Comment (przywartya) [inline tap_tool/__init__.py]: We don't want to remove logging - it's actually quite useful to see in logs that something succeeded. So remove only the slack part :)

Comment (JingLin0) [reply tap_tool/__init__.py]: okay

Comment (JingLin0) [reply tap_tool/__init__.py]: changed

File: tap_tool/__init__.py [hunk 2]
```diff
@@ -125,13 +125,6 @@ def publish_msg(slack, job_failed, company, tap_runtime):
     else:
         success_msg = f"All data for {company} were synced successfully!"
         logging.info(success_msg)
-        if slack:
-            slack_msg(
-                pretext=f"tap-tool succeded for {company}",
-                text=f"{success_msg} \n{tap_runtime}",
-                color="good",
-            )
-        sys.exit()
```
Comment (przywartya) [inline tap_tool/__init__.py]: This too should stay :)

Comment (przywartya) [reply tap_tool/__init__.py]: sys.exit

Comment (JingLin0) [reply tap_tool/__init__.py]: oh yes, just changed

**Pull request merged**